roofit/roofitcore/src/TestStatistics/RooBinnedL.cxx

```diff
@@ -51,6 +50,10 @@ RooBinnedL::RooBinnedL(RooAbsPdf *pdf, RooAbsData *data)
    // The Active label will disable pdf integral calculations
    pdf->setAttribute("BinnedLikelihoodActive");
 
+   RooArgSet* params = pdf->getParameters(data) ;
+   _paramTracker = new RooChangeTracker("chtracker","change tracker",*params,true) ;
+   delete params ;
```

Comment:
Oh, and just below this I see a memory leak of `RooArgSet *obs = pdf->getObservables(data)` because `obs` never gets deleted! I know you have not touched that code in this PR, but can you please fix it too so we won't forget?